.teamcity/settings.kts

```diff
@@ -952,14 +951,88 @@ object RunCalypsoE2eDesktopTests : BuildType({
 				export BROWSERSIZE="desktop"
 				export BROWSERLOCALE="en"
 				export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL%/}\"}"
-				export TEST_FILES=${'$'}(join ',' ${'$'}(ls -1 specs*/**/*spec.js))
+				# Find all spec files but exclude playwright tests.
+				export TEST_FILES=${'$'}(join ',' ${'$'}(find specs*/**/*spec.js -type f -not -path specs-playwright/*))
```

Comment:
It was much easier to achieve the exclusion of `specs-playwright` directory using the `find` command and thus the original `ls -1` call has been replaced.